src/rendition-mixin.js

```diff
@@ -48,11 +48,11 @@ const enableFunction = (loader, playlistUri, changePlaylistFn) => (enable) => {
  */
 class Representation {
   constructor(hlsHandler, playlist, id) {
-    // Get a reference to a bound version of fastQualityChange_
-    let fastChangeFunction = hlsHandler
-                              .masterPlaylistController_
-                              .fastQualityChange_
-                              .bind(hlsHandler.masterPlaylistController_);
+    // Get a reference to a bound version of the quality change function
+    let qualityChangeFunction = (hlsHandler.options_.smoothQualityChange ?
```

Comment:
Personally not a fan of large or multi line ternaries, but feel free to ignore this suggestion as its just my preference.
```
const {
  masterPlaylistController_: mpc,
  options_: { smoothQualityChange }
} = hlsHandler;
const changeType = smoothQualityChange ? 'smooth' : 'fast';
const qualityChangeFunction = mpc[`${changeType}QualityChange_`].bind(mpc);
```

Reply:
Dang, that's much saner. Thanks for the suggestion.